.github/workflows/docs-site.yml

```
@@ -0,0 +1,56 @@
+name: Docs Site
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)